### DISCUSSION TITLE
How to use Jotai with Next.js?

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [pmndrs/jotai#595] [Q&A] [answered]
Hello, I wanted to know if someone had an explanation on how to use Jotai with Next.js 🤔

**Comment by dai-shi** [comment]
cc @Thisen

**Comment by Thisen** [comment, marked as answer]
Hi @MatteoGauthier, please see #340.
Also, there's examples here: https://docs.pmnd.rs/jotai/guides/nextjs

**Reply by MatteoGauthier** [reply]
Okay thanks, i will try. Hope to see an complete example on vercel/next.js

**Reply by dai-shi** [reply]
We have it now: https://github.com/vercel/next.js/tree/canary/examples/with-jotai

**Reply by MatteoGauthier** [reply]
Thanks a lot @dai-shi !
